README.md

```diff
@@ -6,123 +6,16 @@
 
 <br></br>
 
-Rascal is a simple and lightweight [result type](https://www.youtube.com/watch?v=srQt1NAHYC0&t=1018s) implementation for C#, containing a variety of utilities and standard functions for working with result types and integrating them into the rest of C#.
+Rascal is a simple yet powerful [result type](https://www.youtube.com/watch?v=srQt1NAHYC0&t=1018s) implementation for C#, containing a variety of utilities and standard functions for working with result types and integrating them into the rest of C#.
 
 Rascal is first and foremost an aggregate of the result types I personally find myself implementing in a majority of my own projects, and a competetor other result libraries second. As such, this library implements some things I think other result implementations are lacking, while omitting some features other libraries do implement.
 
+**TODO**: Link to the Github Pages docfx documentation.
+
 <br/>
 
 # Installation
 ```ps1
 # .NET CLI
 dotnet add package Rascal --prerelease
 ```
-
-# Using the package
```

Comment:
keep this here. yes, documentation should be done in the docfx, but most people want to know the nuts and bolts right here, without going to the doc page.